loginPrcess.php

```diff
@@ -0,0 +1,38 @@
+<?php
+require 'connection.php';
+
+$userName = $_POST["userName"];
```

Comment:
Don't initialize values like this. First check if those value are assign to the $_POST[] associative array. Values ​​can be assigned to variables only if the condition is true.

Reply:
And "$userName" variable name is not match for logic, this not a issue but best coding practice is we have to follow the propper naming convention.  So, change the variable name.

Reply:
> And "$userName" variable name is not match for logic, this not a issue but best coding practice is we have to follow the propper naming convention. So, change the variable name.

OK